front/app/components/EventPreviews/EventPreviewCard.tsx

```diff
@@ -44,6 +43,22 @@ const StyledEventTitle = styled(Text)`
   -webkit-box-orient: vertical;
 `;
 
+export const EventDate = styled(Box)`
+  display: flex;
```

Comment:
Lifted this out into  styled component as I felt it looked less cluttered like this :thinking: